contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -44,6 +44,18 @@ contract StakeManager is IStakeManager {
         return NFTContract.ownerOf(tokenId);
     }
 
+    function epoch() public view returns(uint256) {
```

Comment:
Any reason for adding a new function?

Reply:
ValidatorShare.sol uses currentEpoch getter which is public getter from StakeManagerStorage. After refactor is no longer presented in IStakeManager and i had to add for compilation reasons.

Reply:
Nice!